src/NuGet.Core/NuGet.Packaging/PackageFolderReader.cs

```diff
@@ -68,7 +70,10 @@ public override string GetNuspecFile()
 
             if (nuspecFiles.Length == 0)
             {
-                throw new PackagingException(Strings.MissingNuspec);
+                var message = new StringBuilder();
+                message.AppendFormat(CultureInfo.CurrentCulture, Strings.Error_MissingNuspecFile);
```

Comment:
`Strings.Error_MissingNuspecFile` doesn't have any `{0}`, so `.Append(...)` can be used for slightly higher performance (the string build doesn't need to parse the string looking for `{`'s.)